desktop-environment.el

```diff
@@ -357,15 +365,21 @@ replacing the placeholder %d with the prefix argument."
 (defun desktop-environment-toggle-mute ()
   "Toggle between muted and un-muted."
   (interactive)
-  (message "%s"
-           (desktop-environment--shell-command-to-string desktop-environment-volume-toggle-command)))
+  (message "Volume %s"
+           (let ((output (desktop-environment--shell-command-to-string desktop-environment-volume-toggle-command)))
+             (save-match-data
+               (string-match desktop-environment-volume-get-state-regexp output)
+               (match-string 0 output)))))
 
 ;;;###autoload
 (defun desktop-environment-toggle-microphone-mute ()
   "Toggle microphone between muted and un-muted."
   (interactive)
-  (message "%s"
-           (desktop-environment--shell-command-to-string desktop-environment-volume-toggle-microphone-command)))
+  (message "Microphone volume %s"
```

Comment:
I'm wondering if "Mic [on]" would be better than "Microphone Volume [on]". What do you think?